src/components/post-card/PostCard.js

```diff
@@ -57,17 +57,24 @@ export function PostCard(props) {
         <div className="card-text body">
           {body}
         </div>
+        <footer className="blockquote-footer">{author}
+        </footer>
       </div>
 
-    {/*  todo: здесь нужно показать массив коментариев к посту (comments), который прилетит в props
+    {/*  donetodo: здесь нужно показать массив коментариев к посту (comments), который прилетит в props
           можно создать отдельную компоненту Comment по аналогии с тем, как мы делали PostCard, UserCard
           как она будет выглядеть зависит от вашей фантазии
           Для каждого комментария должны быть выведены такие поля:
           name - имя автора комментария
           email - почта автора комментария
           body - текст комментария
     */}
-
+      {/*{comment.map(value => {*/}
+      {/*  return(*/}
+      {/*      <Comment key={value.id} comment={value.comment} />*/}
```

Comment:
if I'm not mistaken, there should be `comment={value}`

cause you have array of comments and while map there is each comment taken on each step

and each comment contains all needed information